static/chats.js

```diff
@@ -52,20 +52,23 @@ Klase, kas satur visu vienas ziņas saturu, struktūru un metainformāciju
 Inicializē ar no servera atgrieztā json objekta vienu rindu
 */
 class Zinja {
-  constructor(vards, zinja) {
+  constructor(vards, zinja, laiks) {
     this.vards = vards;
     this.zinja = zinja;
+    this.laiks = laiks;
   }
 
   formateRindu() {
     this.vards = getCookie('name')
+    const laiks = this.laiks ? this.laiks : '-';
     const LIclassName = "left clearfix";
     const newDivclassName = "chat-body clearfix";
     let newLI = document.createElement("li");
     newLI.className = LIclassName;
     let newDiv = document.createElement("div"); 
     newDiv.className = newDivclassName;
-    let teksts = this.vards + ": " + this.zinja;
+    let teksts = `${this.vards}: ${this.zinja}, nosūtīts: ${this.laiks}`;
+    // let teksts = this.vard + ":" + this.zinja + "/nosūtīts: " + this.laiks;
```

Comment:
Šovajag?

Reply:
Nevarēju ar sevi vienoties, kurš pieraksts patīk labāk; strādā abi. Aizkomentēto var dzēst.

Reply:
Pārskatīju šo koda daļu vēlreiz, jo nevarēju saprast, kāpēc vajadzīga rindiņa const laiks = this.laiks ? this.laiks : '-';  ja mainīgais _laiks_ netiek izmantots. 
Izrādās - esmu neuzmanīgi norakstījusi no Slack Tavus ieteikumus (${laiks} vietā ierakstīju ${this.laiks})
Korektāk būtu ${laiks} ?

Reply:
Ja izmanto klases instances mainīgo, tad ar this.
Ja, piemēram, funkcijas ietvaros definējam mainīgo laiks, tad bez this.

Reply:
Un atbildot uz jautājumu jā, tur vajag mainīgo laiks.

Reply:
Kad atrisināsi šo, tad varēs pludināt master.